pages/docs/reference/whatsnew1420.md

```diff
@@ -0,0 +1,382 @@
+---
+type: doc
+layout: reference
+title: "What's New in Kotlin 1.4.20"
+---
+
+# What's New in Kotlin 1.4.0
+
+In Kotlin 1.4.20, we offer a number of new experimental features and provide fixes and improvements for
+existing features, including those added in 1.4.0.
+
+For a detailed explanations of the features listed below, see [this blog post](https://blog.jetbrains.com/kotlin/2020/11/kotlin-1-4-20-released/).
```

Comment:
Let's rephrase this sentence somehow. It looks strange to redirect to the blog post `for a detail explanation of features`. Docs should be the source of detail explanations.

Maybe something like 
> You can also learn about new features with more examples (or what kind of detail?) in the release blog post.